lib/transport/reusable_data/dataset.ex

```diff
@@ -65,6 +65,20 @@ defmodule Transport.ReusableData.Dataset do
           commune_principale: String.t(),
         }
 
+  def regions_lookup, do:  %{"$lookup" => %{
```

Comment:
those are constants, right? is a def the right way to do it?

Reply:
Yep, otherwise it won't be accessible from outside of the module